src/views/GroupSettingsView.java

```diff
@@ -293,6 +312,12 @@ public void propertyChange(PropertyChangeEvent evt) {
             String eventInfo = "<html><b>" + event.getEventName() + "</b><br>Start: " + event.getStartTime().format(formatter) +
                     "<br>End: " + event.getEndTime().format(formatter) + "</html>";
             recommendedEventLabel.setText(eventInfo);
+        } else if ("exportCalendarSuccess".equals(evt.getPropertyName())) {
+            ExportCalendarState exportCalendarState = (ExportCalendarState) evt.getNewValue(); // shows null
+            JOptionPane.showMessageDialog(this, "Group calendar is successfully exported to " + exportCalendarState.getFilePath(), "Export Success", JOptionPane.INFORMATION_MESSAGE);
```

Comment:
In the current implementation, `.getFilePath()` returns `null`. However, since the file path is not something that ever changes, why don't we replace this messageDialog to say something along the lines of "Group calendar is successfully exported to xyzabc folder in LinkUp file directory"?

Where xyzabc is whatever you end up choosing between `Downloads` and `CalendarExports`

Reply:
Just made the change, thanks for the heads up!